doc/reference/annotations.rst

```diff
@@ -127,15 +127,15 @@ to a lower-cased underscored name, e.g. camelCase -> camel_case.
 
 Note that this attribute is not used when you're using any other naming
 strategy than the default configuration (which includes the
-``SerializedNameattributeStrategy``). In order to re-enable the attribute, you
-will need to wrap your custom strategy with the ``SerializedNameattributeStrategy``.
```

Comment:
This appears to be caused by a bad search and replace in 7d7e87d8d4ab7ea3872d1328d4e684a8cab796b8